spark_pipeline_framework/transformers/send_automapper_to_fhir/v1/automapper_to_fhir_transformer.py

```diff
@@ -190,15 +198,27 @@ def _transform(self, df: DataFrame) -> DataFrame:
                     fhir_resource_response_path: str = func_get_response_path(
                         view, resourceType
                     )
+
                     # export as FHIR to local fhir_resource_path
-                    FhirExporter(
-                        view=view,
-                        file_path=fhir_resource_path,
-                        parameters=parameters,
-                        progress_logger=progress_logger,
-                        name=f"{name}_fhir_exporter",
-                        mode=mode,
-                    ).transform(df)
+                    if file_format == "parquet":
+                        FrameworkParquetExporter(
+                            name=f"{name}_parquet_exporter",
+                            parameters=parameters,
+                            progress_logger=progress_logger,
+                            view=view,
+                            mode=mode,
+                            file_path=fhir_resource_path,
+                        ).transform(df)
```

Comment:
I found that there is already , `FrameworkParquetExporter` transformer, so just used it here when it's parquet